dashboard/db/migrate/20170919054651_create_school_yearly_stats.rb

```diff
@@ -0,0 +1,51 @@
+class CreateSchoolYearlyStats < ActiveRecord::Migration[5.0]
+  def change
+    create_table :school_yearly_stats do |t|
```

Comment:
I'd prefer a different name for this table. I would expect `school_yearly_stats` to contain stats about schools for 2017, 2016, etc. I get that this title was about grade years, but already this table contains data beyond grade years. How about `school_metadata`?

Reply:
It's more than metadata, however, there are counts and ben is thinking about adding additional rollups. Any other name suggestions?

Reply:
Yeah I don't think the title is about grade years -- it was about things that change yearly (e.g., student population, etc.)

Reply:
Alternatively, `school_yearly_infos`, or `school_yearly_surveys`?

I'd like to keep yearly in the name so that we understand that the change is based on school year. The `school_year` field would contain values like: 2013-2014, 2014-2015, 2015-2016, 2016-2017. This value corresponds to the survey year in the NCES source data.